app/assets/stylesheets/application.css.scss

```diff
@@ -51,3 +51,33 @@ path {
   stroke: #fff;
   fill-rule: evenodd;
 }
+
+.d3-tip {
+ line-height: 1;
+ font-weight: bold;
+ padding: 12px;
+ background: rgba(0, 0, 0, 0.8);
+ color: #fff;
+ font-color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Line should be indented 2 spaces, but was indented 1 spaces<br>Unknown property font-color